docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -0,0 +1,48 @@
+html {
+  font-size: 16px;
```

Comment:
We don't need to specify a font-size here, 16px is the default in almost all browsers. Not setting a default also prevents us from overriding a users custom browser font-size which they may have set for accessibility reasons. Letting them set their own font-size and then using rem units throughout the site lets the page scale gracefully.